### PR TITLE
Add mini test

### PR DESCRIPTION
**Pull request by stan-is-hate**
This is a super simple test that verifies zk and kafka can start up.
The purpose is mostly to verify any changes to ducktape, ducker-ak or underlying test infrastructure, but it is also useful for onboarding new engineers.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by stan-is-hate** [comment]
@omkreddy @lbradstreet

File: tests/kafkatest/tests/core/mini_test.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class MiniTest(Test):
+    def __init__(self, test_context):
+        super(MiniTest, self).__init__(test_context=test_context)
+
+        self.zk = ZookeeperService(test_context, 1)
+        self.kafka = KafkaService(test_context, 1, self.zk)
+
+    @cluster(num_nodes=2)
+    def test(self):
+        self.zk.start()
```
Comment (hachikuji) [inline tests/kafkatest/tests/core/mini_test.py]: If it's just as well, could we use kraft instead, or alternatively provide two parameterizations?

Comment (stan-is-hate) [reply tests/kafkatest/tests/core/mini_test.py]: Yeah, sorry, this slipped my radar. It's a good idea, I'll add it - but how can we push this PR to older versions if it has kraft? Would it have to be a separate PR for older versions?

**Comment by stan-is-hate** [comment]
Passed: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/5019/

File: tests/kafkatest/tests/core/mini_test.py (hunk 1 above)
Comment (stan-is-hate) [reply tests/kafkatest/tests/core/mini_test.py]: Added kraft - I'm not super familiar with how kraft support is implemented in kafka tests, so please let me know if this is the right way to do it. Tests passed fwiw.

**Pull request closed without merging**